From 64ce2e502e3794f1da9a7cd44b00e1ac4c6a84b4 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Wed, 6 Mar 2024 18:12:36 +0800
Subject: [PATCH] fix(org.kde.ksystemlog.desktop): drop kdesu marker

kdesu behavior may appear erratic to non-admin users, don't bother asking.
---
 src/org.kde.ksystemlog.desktop | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/org.kde.ksystemlog.desktop b/src/org.kde.ksystemlog.desktop
index 09e00a6..8b249a1 100644
--- a/src/org.kde.ksystemlog.desktop
+++ b/src/org.kde.ksystemlog.desktop
@@ -4,7 +4,6 @@ Exec=ksystemlog -qwindowtitle %c
 Icon=utilities-log-viewer
 Categories=Qt;KDE;System;Monitor;
 X-DocPath=ksystemlog/index.html
-X-KDE-SubstituteUID=true
 Keywords=log;logfile;system messages;system log;kernel log;authentication log;X.org log;journald log;services logs;Event Viewer;eventvwr;eventlog;
 Keywords[ar]=سجل;رسائل النظام;رسائل;سجل النواة;سجل خادم العرض;عارض الأحداث;سجل الخدمات;
 Keywords[az]=jurnal;jurnal faylı;sistem ismarıcı;sistem jurnalı;nüvə jurnalı;kimlik doğrulaması jurnalı;X.org jurnalı;journald jurnalı;xidmətlər jurnalı;Hadisələrə Baxış;eventvwr;hadisələr jurnalı;
-- 
2.44.0

